packages/aws-iam-authenticator/aws-iam-authenticator.spec

```
@@ -48,6 +48,7 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)
 
 %build
 %set_cross_go_flags
+export GO_VERSION="1.22.4"
```

This means the same thing as of the latest SDK release, and will help avoid stale `GO_VERSION` entries in spec files (or busywork commits to update them all whenever we bump Go):
```suggestion
export GO_MAJOR="1.22"
```